src/router.js

```diff
@@ -17,13 +17,17 @@ const { bifibuyback } = require('./api/stats/bifibuyback/index');
 const { getTokens, getChainTokens } = require('./api/tokens');
 const { getConfigs, getChainConfig } = require('./api/config');
 const { getTreasury } = require('./api/treasury');
+const { validatorPerformance } = require('./api/validators/index');
+
 const {
   vaultZapSupport,
   vaultZapSupportDebug,
   proxyOneInchSwap,
   proxyOneInchQuote,
 } = require('./api/zaps');
 
+router.get('/validator/performance', validatorPerformance);
```

Comment:
It looks weird to me for us to support `/validator/performance` when we don't support `/validator`.

Reply:
What's a solution to this?  Or i can have it /validators, and can add support for /validators to return just the validators i guess

Reply:
Yeah maybe `/validator` or `/validator-performance`